### DISCUSSION TITLE
Converting from container variables to .env file.

### DISCUSSION THREAD

**Discussion by devanteweary** [luigi311/JellyPlex-Watched#131] [Q&A]
Hey guys,

Using the Unraid Docker container here.

Just wondering if I'd be able to convert from the container variables to a config.env in the /appdata/jellyplex-watched/ directory.

I'm thinking I can delete all the variables in the container and somehow point it to that config.env file.

The reason is just makes it more convenient for me to add users and I might even be able to piece together a script that you can just enter the two names and it adds them to the config.env file. But that's another story. Trying to learn C# currently!

I also like the portability of a config file.

Alrighty thanks for any info guys!

**Comment by Guruleenyc** [comment]
. 

> Hey guys,
> 
> Using the Unraid Docker container here.
> 
> Just wondering if I'd be able to convert from the container variables to a config.env in the /appdata/jellyplex-watched/ directory.
> 
> I'm thinking I can delete all the variables in the container and somehow point it to that config.env file.
> 
> The reason is just makes it more convenient for me to add users and I might even be able to piece together a script that you can just enter the two names and it adds them to the config.env file. But that's another story. Trying to learn C# currently!
> 
> I also like the portability of a config file.
> 
> Alrighty thanks for any info guys!

Ditto! Agreed here as well

**Comment by luigi311** [comment]
while working on another project i found a different function to load the environment variables that will let me specify a file to read from. I can bring that in here and take a look at setting up unraid to use that but i need to design it in a way that doesnt break the previous method of setting environment variables.

**Comment by luigi311** [comment]
converted to issue at #260
